src/ipld/util.rs

```diff
@@ -195,12 +200,12 @@ impl<DB: Blockstore, T: Borrow<Tipset>, ITER: Iterator<Item = T> + Unpin> Stream
                                 return Poll::Ready(Some(Ok(CarBlock { cid, data })));
                             } else if fail_on_dead_links {
                                 return Poll::Ready(Some(Err(anyhow::anyhow!(
-                                    "missing key: {cid}"
+                                    "[Emit] missing key: {cid}"
                                 ))));
                             };
                         }
                     }
-                    Iterate(cid_vec) => {
+                    Iterate(epoch, block_cid, ty, cid_vec) => {
```

Comment:
what's `ty`? let's use a more meaningful name.

Reply:
It's `type` but reserved as keyword. How about `_type`?